src/services/SessionRequestProcessor.ts

```diff
@@ -208,16 +207,18 @@ export class SessionRequestProcessor {
   		return Response(HttpCodesEnum.SERVER_ERROR, "Internal server error");
   	}
 
-  	if (jwtPayload.shared_claims) {
-  		try {
-  			await this.f2fService.savePersonIdentity(jwtPayload.shared_claims, sessionId);
-  		} catch (error) {
-  			this.logger.error("Failed to create session in person identity table", {
-  				error,
-  				messageCode: MessageCodes.FAILED_SAVING_PERSON_IDENTITY,
-  			});
-  			return Response(HttpCodesEnum.SERVER_ERROR, "Internal server error");
-  		}
+  	try {
+  		// If multiple addresses present, retrieve preferred address from shared_claims
```

Comment:
Given how explicit the code is here, is it more helpful to either update the comment, or add a doc comment onto the function to describe what 'preferred address' means (seemingly current address?)